x-pack/plugins/gis/public/shared/layers/sources/ems_file_source/ems_file_source_source_editor.js

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Can you rename this to be more consistent with the other sources to something like `create_source_editor.js`